tools/devtool

```diff
@@ -1199,7 +1193,7 @@ cmd_install() {
     # Install the binaries
     for binary in "${binaries[@]}"; do
         say "Installing $binary in $install_path"
-        install -m 755 "$( build_bin_path "$target" "$profile" "$binary" )" "$install_path"
+        install -m 755 "$( build_bin_path "$target" "$profile" "$binary" )" -Dt "$install_path"
```

Comment:
nit: I'd put the `-Dt` before the file argument, so that all options are at the beginning, and then all the arguments.

Reply:
this could look like:

```-Dtm755``` or ```-D -t -m 755```

any preference?

Reply:
I just matched the order in the man file. Change reflected in the commit.

Reply:
Thanks!